spec/factories/user.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :user do
+    email 'test@example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.